types/cytoscape/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for Cytoscape.js 3.4
+// Type definitions for Cytoscape.js 3.8
```

Comment:
So the new values in `ArrowShape` were not valid below cytoscape@3.8? The version string should be the _lowest_ version that these types are valid for.

Reply:
it is a pity, that they dont's maintain CHANGELOG file and I don't have enough time right now to scan branches of sources for lowest version where these edge constants are available. At least, they are in 3.8 for sure.

Reply:
Fair enough 👍